### ISSUE TITLE
Feature / shortcut request: Keyboard shortcut to focus on the search textbox (or just search)

### ISSUE THREAD

**Issue by WaterSibilantFalling** [pawelsalawa/sqlitestudio#4598]
I am often, often searching for a text string across the displayed rows. 

For me, and others, it would be great if there was (at least an option to have) a way to use only the keyboard to search.       

![image](https://user-images.githubusercontent.com/8019455/204685315-87728064-ec08-4b78-9585-a6cd802aae1c.png)

Currently AFAIK there is no  only-keyboard way to tab-tab - tab -keystroke - tab . . . . .  to the search . You have to use the mouse if you want to search for a text string.

**Comment by JrgMyr** [comment]
You mean like <Ctrl-F> in table view?

Should that trigger the already existing filter function (as seen in your snapshot) or work more like <Ctrl-F> in the query view?

**Comment by WaterSibilantFalling** [comment]
Yes (unless I misunderstand the view names), in table view. In the view shown when I click on a table.   

![image](https://user-images.githubusercontent.com/8019455/205153411-a7b9e98a-ba37-4d2a-8ad0-17a73c04cc5e.png)

Ah yes, like what is available when editing queries.  

![image](https://user-images.githubusercontent.com/8019455/205154286-5da2c62c-129e-43fe-9b55-1bf7f58a50ac.png)

If (by default) Ctrl-F was mapped to "(table-view) select the search box" or at least available to be mapped to the table-view search box, that would be great. 

------

Possibly a bigger or more foundational question: would it be possible to have _every_ GUI element potentially mappable to a keystroke.  Different activities will be common to different people - for me, it's fixing up data - so different people would love to have who-knows-what different GUI elements (at least potentially) quickly actionable via keyboard shortcuts.

**Comment by WaterSibilantFalling** [comment]
> Should that trigger the already existing filter function (as seen in your snapshot) or work more like in the query view?

To be clear:

Yes:  trigger the already existing filter function (as seen in your snapshot) 

Yes: the same keystoke as in the query view, the same Ctrl-F to do a find, would be good

No: I wasn't suggesting a 'find individual text string' and optionally edit it, but that could be good functionality for some use cases.  

I was thinking about selecting, filtering rows. 

Good call: there are two different meanings for "find" when looking at table data (1: rows, 2: sub-row text values). For blocks of text, where 'rows' aren't a relevant concept, there is only one meaning for "find". 

Possibly   
* Ctrl-F to directly select a text/field value, so the idea and functionality maps the existing query-edit functionality as close as possible. Extra good if only searched in currently displayed rows (so can first filter on containing "AAA", and then, of these, search-replace text-like "bbb" into "ccc". 
* Ctrl-shift-F (say) to open the filter-text box, as it is _not_ exactly the same functionality as in the query text-block search.

Anyway, my idea/request was to have it possible, somehow, to select the row-filter box by using a keyboard shortcut.  Then, with a certain number of "tab" pushes, I can tab through to the N-th column and edit the data in row 1 . . .

**Comment by WaterSibilantFalling** [comment]
looking at #4599 , that's a third search option: look for a text string across . . . . everything. That not my use case, but for someone else. 

My request is really to have (the possibility) of selecting a GUI item - which happens to be a search box - by a keyboard shortcut.
